src/clients/cassandra/CassandraConnector.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Promise = require('promise');
+
+module.exports = {
```

Comment:
yes I was wondering if it would be better at the top so you can quickly glance and see all the functions that you would have available to you.